packages/cli/src/commands/start.ts

```diff
@@ -24,17 +18,19 @@ export class Start extends Command {
       char: "H",
       description: "Set server hostname",
     }),
+    ts: flags.boolean({
+      description: "Use typescript explicitly (just for performance optimization)",
+    }),
```

Comment:
Is this really actually necessary? Seems like a slightly redundant feature when we are only avoiding a fs read at the start of a cli process

Reply:
The main reason was to find a way to avoid `fs.existsSync` called in config.ts which just checks if `tsconfig.json` file exists. I was thinking like following:

- As a newcomer to blitz I want to create and start my application with almost zero-config so I run `blitz start` and everything works as expected. 
- Once I settled my initial code base (js, ts) I want to avoid any unnecessary "delays" in dev process. So I have got an option to tell blitz "hey I am using TS, do not check it on your own" by running `blitz start --ts`.

Also this flag can be used in different commands (generate, console) to avoid checking ts implicitly. But honestly I did not make any perf tests and cannot say now if it is worth. So I do not have any compelling reason to keep this flag right now.

Reply:
I think there are other way more expensive things we are doing as the command starts up that are going to be slower than one file stats read especially if we do any file pipeline work.

Reply:
Removed in current commit